### DISCUSSION TITLE
AZURE problem

### DISCUSSION THREAD

**Discussion by nourallouche** [community/community#60511] [GitHub Education]
### Hi everyone,

I’m a Github student developer pack member, and I’m trying to activate my azure account but I have not received any verification code. Please provide me the Verification Code.

**Comment by samus-aran** [comment]
Hi @nourallouche 👋 

If this [Azure verification page](https://signup.azure.com/studentverification?offerType=1) does not work then we would suggest you reach out to [Azure Support](https://support.microsoft.com/en-us/topic/2315e669-8b1f-493b-5fb1-d88a8736ffe4) if you are having problems redeeming your Azure benefits.

**Reply by Akhilrajan1997** [reply]
What can I expect when I contact them? What will happen then? Will they activate it for me, or provide alternate means or what exactly? Is deleting my Microsoft account and creating it again a solution worth trying?
